examples/set-namespace-defaultns-mode/README.md

```diff
@@ -0,0 +1,55 @@
+# set-namespace: Defaultns Mode Example
+
+### Overview
+
+This example demonstrates how to run [`set-namespace`] function in defaultns mode. Only the namespace field
```

Comment:
> in defaultns mode

We should link it to the related section in the usage doc.

Reply:
Done. Also bump the version to v0.3